client/me/help/help-courses/course.jsx

```diff
@@ -0,0 +1,24 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+import Card from 'components/card';
+
+export class Course extends Component {
+	render() {
+		return (
+			<Card className="help-courses__course">
+				Latest course: Only visible for users with the business plan
+			</Card>
+		);
+	}
+}
+
+export const CoursePlaceholder = () => {
+	return (
+		<Card className="help-courses__course is-placeholder"></Card>
```

Comment:
I wonder if instead of having a separate component just for rendering a placeholder, we could move this line to `Course` if a prop `isPlaceholder` is `true`. That way, we could then import `Course` and render it with `isPlaceholder={ isPlaceholder }` prop instead of having to import a separate component.

This would also move the placeholder logic from `Courses` component to the specific components and simplify it.

Don't forget about doing the same for `CourseList` if you decide to go this route. :)